app/services/lara_serialization_helper.rb

```diff
@@ -21,6 +21,18 @@ def export(item)
       # interactive missing. I guess it could be double checked and fixed in the future.
       results[:linked_interactive] = self.export(item.linked_interactive)
     end
+
+    # This is done here and not in the models as we don't want to export the linked embeddables as they are already exported
+    # we only want to save the key so that the references to the embeddables can be restored during import.
+    # The embeddedable references are saved instead of the page_item references as new page_items are created when the
+    # embeddables are imported into the page.
+    if item.respond_to?(:primary_linked_items)
+      results[:linked_page_items] = item.primary_linked_items.map {|pli| {
```

Comment:
Since this export is what is used by the activity player as its activity structure format. I think it is worth keeping it more independent of the implementation.

So I'd recommend changing this from `results[:linked_page_items]` to `results[:linked_interactives]` I know this is going to be somewhat confusing since there is also linked_interactive. But that is the way it is called in the new field of the initInteractive message, so using the same name here seems better.

Additionally the `primary_ref_id` doesn't seem necessary, and would likely confuse a developer working with the export.  Without the primary_ref_id, then the `secondary_ref_id` could be changed to `interactive_ref_id` which would be consistent with other references to interactives.